pkg/config/env/cdn.go

```diff
@@ -0,0 +1,23 @@
+package env
+
+import (
+	"net/url"
+
+	"github.com/gobuffalo/envy"
+)
+
+// CdnEndpointWithDefault returns CDN endpoint if set
+// if not it should default to clouds default blob storage endpoint e.g
+func CdnEndpointWithDefault(value *url.URL) *url.URL {
```

Comment:
can you use `CDN` instead of `Cdn` here? that's a go convention to upper-case all letters in an acronym